Setup/DeployPrivateKudu.cmd

```diff
@@ -1,48 +1,58 @@
-@if "%_echo%" neq "1" @echo off
```

Comment:
It is not related to this Ssl validation.  I will separate to another commit.   

As for what it was about, this is the script helping installing private kudu to a site.  Currently we look for curl.exe at `%ProgramFiles(x86)%\git\bin\curl.exe`, and if not exists, `%ProgramW6432%\git\mingw64\bin\curl.exe`.  I added one extra location `%ProgramFiles(x86)%\git\mingw32\bin\curl.exe`.   This would support people installing Git either 32bits or 64bits favors.